Find a functionally equivalent code:android.bluetooth.BluetoothInputDevice.setReport:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * Send Set_Report command to the connected HID input device.
 *
 * <p>Requires {@link android.Manifest.permission#BLUETOOTH_ADMIN} permission.
 *
 * @param device Remote Bluetooth Device
 * @param reportType Report type
 * @param report Report receiving buffer size
 * @return false on immediate error,
 * true otherwise
 * @hide
 */

@@ -1,5 +1,5 @@
 {
-    if (DBG)
+    if (VDBG)
         log("setReport(" + device + "), reportType=" + reportType + " report=" + report);
     if (mService != null && isEnabled() && isValidDevice(device)) {
         try {

